<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TripleStoreDialog</class>
 <widget class="QDialog" name="TripleStoreDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>267</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>638</width>
    <height>267</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QObject#TripleStoreDialog{
background-color:rgb(255,255,255)
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>541</width>
       <height>171</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QObject#frame{
background-color: rgb(140, 140, 140);
	
Border-radius:20px;}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="tripleStoreNameLabel">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Triple Store Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="tripleStoreNameEdit">
            <property name="minimumSize">
             <size>
              <width>401</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius:10px;

border-color:rgb(45, 45, 45);

background-color: rgb(253, 252, 250);</string>
            </property>
            <property name="text">
             <string>My cool triplestore!</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="tripleStoreLabel">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Triple Store URL:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="tripleStoreEdit">
            <property name="minimumSize">
             <size>
              <width>401</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius:10px;

border-color:rgb(45, 45, 45);

background-color: rgb(253, 252, 250);</string>
            </property>
            <property name="text">
             <string>https://digits.mainzed.org/pleiades/sparql</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="detectConfiguration">
          <property name="minimumSize">
           <size>
            <width>191</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(189, 24, 29);
border-radius:10px;
color: rgb(255, 255, 255);
</string>
          </property>
          <property name="text">
           <string>Quick Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="tripleStoreCloseButton">
          <property name="minimumSize">
           <size>
            <width>181</width>
            <height>23</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(189, 24, 29);
border-radius:10px;
color: rgb(255, 255, 255);
</string>
          </property>
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="permanentAdd">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Add triple store permanently</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="tripleStoreNameLabel_2">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>                Enter a name and a triple store URL here to try autoconfiguration</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
